### PR TITLE
Add target arch support

### PR DESCRIPTION
**Pull request by thughes**
Addresses comments in pull request #55.

### CODE REVIEW

**Comment by leshow** [comment]
Please merge this!

File: install.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var paths = {
 if (!paths[platform]) throw new Error('Unknown platform: ' + platform)
 
 // downloads if not cached
-download({version: version}, extractFile)
+download({version: version, arch: process.env.npm_config_arch}, extractFile)
```
Comment (maxogden) [inline install.js]: sorry to nitpick, but this should be `ELECTRON-ARCH` (in uppercase)

Comment (thughes) [inline install.js]: The environment variable? `npm_config_arch` is automatically set when using the `--arch` flag with `npm`, which is why I used it (see docs I added).

Comment (maxogden) [inline install.js]: Ohh right I forgot npm did that automatically now, sorry. Looks good to me then

**Comment by maxogden** [comment]
this will get published the next time electron gets released by GitHub

**Pull request merged**